src/CMakeLists.txt

```diff
@@ -631,6 +634,16 @@ if(NOT WIN32 AND NOT APPLE)
     endif()
 endif()
 
+############################################################
+# MIOpen depends on libMLIRMIOpen for implicit gemm
+if(LIBMLIRMIOPEN AND NOT MIOPEN_USE_COMGR)
+  MESSAGE(STATUS "LibMLIRMIOpen: " ${LIBMLIRMIOPEN})
+  set(MIOPEN_LIBMLIRMIOPEN "${LIBMLIRMIOPEN}")
```

Comment:
@pfultz2  FYI two cmake variables:

- LIBMLIRMIOPEN for tracking if library exists
- MIOPEN_LIBMLIRMIOPEN for macro used in code